xl-erasure-v1-errors.go

```diff
@@ -19,10 +19,13 @@ package main
 import "errors"
 
 // errMaxDisks - returned for reached maximum of disks.
-var errMaxDisks = errors.New("Total number of disks specified is higher than supported maximum of '16'")
+var errMaxDisks = errors.New("Number of disks on command line is higher than supported maximum of '16'")
+
+// errMaxDisks - returned for minimum number of disks.
+var errMinDisks = errors.New("Number of disks on command line is smaller than supported minimum of '8'")
 
 // errNumDisks - returned for odd numebr of disks.
-var errNumDisks = errors.New("Invalid number of disks provided, should be always multiples of '2'")
+var errNumDisks = errors.New("Invalid number of disks command line, should be always multiples of '2'")
```

Comment:
Its better not to mention "command line" here.  We could say something like "Number of disks should be multiples of 2"